lib/utils/config.ts

```diff
@@ -86,24 +87,7 @@ export async function generateBaseConfig(
 	return config;
 }
 
-export async function generateApplicationConfig(
-	application: BalenaSdk.Application,
-	options: {
-		version: string;
-		deviceType?: string;
-		appUpdatePollInterval?: number;
-	},
-) {
-	const config = await generateBaseConfig(application, options);
-
-	if (semver.satisfies(options.version, '<2.7.8')) {
-		await addApplicationKey(config, application.id);
-	} else {
-		await addProvisioningKey(config, application.id);
-	}
-
-	return config;
-}
+export const generateApplicationConfig = generateBaseConfig;
```

Comment:
@thgreasi Is the `config.apiKey` from `generateBaseConfig` sufficient? Or to put it another way, the version `2.7.8` bifurcation isn't necessary for `generateApplicationConfig`?

I don't have history on this version change, that is why the ask.

Reply:
The API (and as a result `os.getConfig()`) already handles that case and creates the correct type of apiKey depending on the version.